pkg/oc/cli/newapp/newapp.go

```diff
@@ -47,7 +47,7 @@ import (
 	"github.com/openshift/origin/pkg/cmd/util/print"
 	imageapi "github.com/openshift/origin/pkg/image/apis/image"
 	imageclientinternal "github.com/openshift/origin/pkg/image/generated/internalclientset"
-	"github.com/openshift/origin/pkg/oc/lib/newapp"
+	generate "github.com/openshift/origin/pkg/oc/lib/newapp"
```

Comment:
Where is this coming from? Feels like it would break compilation

Reply:
ah...  the files in oc/lib/newapp are actually in the "generate" package.  who the heck.......

Reply:
yep that's right re: https://github.com/openshift/origin/pull/22654#issuecomment-486912132